### ISSUE TITLE
i use for my 1.7PM server when player joined and kicked by suuden"internal server erro"

### ISSUE THREAD

**Issue by w2333333** [poggit-orphanage/EconomyS#35]
#0 EconomyAPI-master/src/onebone/economyapi/EconomyAPI(112): onebone\economyapi\EconomyAPI->createAccount(string w2333333, integer 1000, boolean 1)
[01:26:57] [Server thread/DEBUG]: #1 src/pocketmine/plugin/MethodEventExecutor(38): class@anonymous/home/gs/gs/data/servers/11001606/plugins/EconomyAPI-master/src/onebone/economyapi/EconomyAPI.php0x7f31c9762d13->handlePlayerJoin(pocketmine\event\player\PlayerJoinEvent object)
[01:26:57] [Server thread/DEBUG]: #2 src/pocketmine/plugin/RegisteredListener(98): pocketmine\plugin\MethodEventExecutor->execute(class@anonymous./home/gs/gs/data/servers/11001606/plugins/EconomyAPI-master/src/onebone/economyapi/EconomyAPI.php0x7f31c9762d13 object, pocketmine\event\player\PlayerJoinEvent object)
[01:26:57] [Server thread/DEBUG]: #3 src/pocketmine/event/Event(96): pocketmine\plugin\RegisteredListener->callEvent(pocketmine\event\player\PlayerJoinEvent object)
[01:26:57] [Server thread/DEBUG]: #4 src/pocketmine/Player(980): pocketmine\event\Event->call()
[01:26:57] [Server thread/DEBUG]: #5 src/pocketmine/network/mcpe/handler/PreSpawnSessionHandler(106): pocketmine\Player->doFirstSpawn()
[01:26:57] [Server thread/DEBUG]: #6 src/pocketmine/network/mcpe/protocol/SetLocalPlayerAsInitializedPacket(45): pocketmine\network\mcpe\handler\PreSpawnSessionHandler->handleSetLocalPlayerAsInitialized(pocketmine\network\mcpe\protocol\SetLocalPlayerAsInitializedPacket object)
[01:26:57] [Server thread/DEBUG]: #7 src/pocketmine/network/mcpe/NetworkSession(207): pocketmine\network\mcpe\protocol\SetLocalPlayerAsInitializedPacket->handle(pocketmine\network\mcpe\handler\PreSpawnSessionHandler object)
[01:26:57] [Server thread/DEBUG]: #8 src/pocketmine/network/mcpe/NetworkSession(191): pocketmine\network\mcpe\NetworkSession->handleDataPacket(pocketmine\network\mcpe\protocol\SetLocalPlayerAsInitializedPacket object)
[01:26:57] [Server thread/DEBUG]: #9 src/pocketmine/network/mcpe/RakLibInterface(143): pocketmine\network\mcpe\NetworkSession->handleEncoded(string x.c*d.....u)
[01:26:57] [Server thread/DEBUG]: #10 vendor/pocketmine/raklib/src/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 23.242.251.205 49550, raklib\protocol\EncapsulatedPacket object, integer 0)
[01:26:57] [Server thread/DEBUG]: #11 src/pocketmine/network/mcpe/RakLibInterface(88): raklib\server\ServerHandler->handlePacket()
[01:26:57] [Server thread/DEBUG]: #12 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
[01:26:57] [Server thread/DEBUG]: #13 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
[01:26:57] [Server thread/DEBUG]: #14 src/pocketmine/Server(2310): pocketmine\snooze\SleeperHandler->sleepUntil(double 1540272417.7103)
[01:26:57] [Server thread/DEBUG]: #15 src/pocketmine/Server(2185): pocketmine\Server->tickProcessor()
[01:26:57] [Server thread/DEBUG]: #16 src/pocketmine/Server(1728): pocketmine\Server->start()
[01:26:57] [Server thread/DEBUG]: #17 src/pocketmine/PocketMine(251): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gs/gs/data/servers/11001606/, string /home/gs/gs/data/servers/11001606/plugins/)
[01:26:57] [Server thread/DEBUG]: #18 /home/gs/gs/data/servers/11001606/pm.phar(1): require(string phar:///home/gs/gs/data/servers/11001606/pm.phar/src/pocketmine/PocketMine.php)
[01:26:57] [Server thread/INFO]: w2333333 left the game
[01:26:57] [Server thread/INFO]: w2333333[/23.242.251.205:49550] logged out due to Internal server error

**Comment by w2333333** [comment]
please fix this

**Comment by Awzaw** [comment]
I can't reproduce this error, and you didn't give me much information to work with to help you. Are you using the official PocketMine 3.3.0 release and EconomyAPI 5.7.2? Was it working before you updated PocketMine? Which provider do you use (yaml/mysql) and if using mysql are you 100% sure your settings are correct?

**Comment by w2333333** [comment]
yes, I’m using the newest pocketmineMP for minecraft 1.7 version I also tried the newest api you released, but looks need api 4.0.0 and plugin’s api is 3.0,0

**Comment by w2333333** [comment]
nothing happene when i type command, but the plugin is load
[03:08:58] [Server thread/INFO]: Enabling EconomyAPI v2.0.9
[03:08:58] [Server thread/INFO]: [EconomyAPI] Language set to User Defined by onebone
[03:08:58] [Server thread/CRITICAL]: ReflectionException: "Class pocketmine\event\server\ServerCommandEvent does not exist" (EXCEPTION) in "src/pocketmine/plugin/PluginManager" at line 571
[03:08:58] [Server thread/DEBUG]: #0 src/pocketmine/plugin/PluginManager(571): ReflectionParameter->getClass()
[03:08:58] [Server thread/DEBUG]: #1 EconomyAPI/src/onebone/economyapi/EconomyAPI(184): pocketmine\plugin\PluginManager->registerEvents(onebone\economyapi\EconomyAPI object, onebone\economyapi\EconomyAPI object)
[03:08:58] [Server thread/DEBUG]: #2 src/pocketmine/plugin/PluginBase(117): onebone\economyapi\EconomyAPI->onEnable()
[03:08:58] [Server thread/DEBUG]: #3 src/pocketmine/plugin/PluginManager(415): pocketmine\plugin\PluginBase->onEnableStateChange(boolean 1)
[03:08:58] [Server thread/DEBUG]: #4 src/pocketmine/Server(1971): pocketmine\plugin\PluginManager->enablePlugin(onebone\economyapi\EconomyAPI object)
[03:08:58] [Server thread/DEBUG]: #5 src/pocketmine/Server(1957): pocketmine\Server->enablePlugin(onebone\economyapi\EconomyAPI object)
[03:08:58] [Server thread/DEBUG]: #6 src/pocketmine/Server(1726): pocketmine\Server->enablePlugins(integer 1)
[03:08:58] [Server thread/DEBUG]: #7 src/pocketmine/PocketMine(251): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gs/gs/data/servers/11001606/, string /home/gs/gs/data/servers/11001606/plugins/)
[03:08:58] [Server thread/DEBUG]: #8 /home/gs/gs/data/servers/11001606/pm.phar(1): require(string phar:///home/gs/gs/data/servers/11001606/pm.phar/src/pocketmine/PocketMine.php)
[03:08:58] [Server thread/INFO]: Disabling EconomyAPI v2.0.9
